src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -1280,6 +1286,43 @@ internal static int GetBinIdx(float output, float min, float binSize, int numBin
             return binIdx;
         }
 
+        bool ISingleCanSaveOnnx.SaveAsOnnx(OnnxContext ctx, string[] outputNames, string featureColumn)
+        {
+            _host.CheckValue(ctx, nameof(ctx));
+            _host.CheckValue(outputNames, nameof(outputNames));
+            _host.Check(Utils.Size(outputNames) == 2);
+
+            const int minimumOpSetVersion = 9;
+            ctx.CheckOpSetVersion(minimumOpSetVersion, "NaiveCalibrator");
+
+            string opType = "Sub";
+            var minVar = ctx.AddInitializer(Min, "Min");
+            var subNodeOutput = ctx.AddIntermediateVariable(NumberDataViewType.Single, "subNodeOutput");
+            var node = ctx.CreateNode(opType, new[] { outputNames[0], minVar }, new[] { subNodeOutput }, ctx.GetNodeName(opType), "");
+
```

Comment:
For the sake of readability, can you please add an assert above on what you expect to find the in outputNames[0] ?

Reply:
Hi Harish,
There is no reference to the actual name of the score column name, which is what outputNames[0] refers to. Below is where NaiveCalibrator's `ISaveAsOnnx.SaveAsOnnx` is called:
https://github.com/dotnet/machinelearning/blob/bc10d609cc11dbf2c1aafaaf32ed25c6867e4a66/src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs#L265-L277
The name of the score column may differ from its default value of "Score". As seen above, the `outputNames[0]` will always be equal to the name of the score column. I will add comments explaining this.

Reply:
That sounds good.

---
In reply to: [453029703](https://github.com/dotnet/machinelearning/pull/5289#discussion_r453029703) [](ancestors = 453029703)